package.json

```diff
@@ -1,46 +1,46 @@
 {
-    "name": "dominators-phantom-fe",
-    "version": "1.0.0",
-    "description": "dream phantom project",
-    "main": "index.js",
-    "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "start": "webpack-dev-server --mode development --open ",
-        "build": "webpack --mode production"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/atlp-rwanda/Dominators-Phantom-FE.git"
-    },
-    "author": "atlp-dominators",
-    "license": "ISC",
-    "bugs": {
-        "url": "https://github.com/atlp-rwanda/Dominators-Phantom-FE/issues"
-    },
-    "homepage": "https://github.com/atlp-rwanda/Dominators-Phantom-FE#readme",
-    "dependencies": {
-        "react": "^18.0.0",
-        "react-dom": "^18.0.0",
-        "react-icons": "^4.3.1",
-        "react-redux": "^7.2.8",
-        "react-router-dom": "^6.3.0",
-        "react-scripts": "^5.0.0",
-        "react-toastify": "^8.2.0",
-        "redux": "^4.2.0"
-    },
-    "devDependencies": {
-        "@babel/core": "^7.17.9",
-        "@babel/preset-env": "^7.16.11",
-        "@babel/preset-react": "^7.16.7",
-        "babel-loader": "^8.2.4",
-        "css-loader": "^5.2.7",
-        "dotenv": "^16.0.0",
-        "file-loader": "^6.2.0",
-        "html-webpack-plugin": "^4.5.2",
-        "style-loader": "^2.0.0",
-        "webpack": "^4.46.0",
-        "webpack-cli": "^4.9.2",
-        "webpack-dev-server": "^4.8.1"
-    }
+  "name": "dominators-phantom-fe",
+  "version": "1.0.0",
+  "description": "dream phantom project",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack --config webpack.prod.js",
```

Comment:
Also here, it looks like you are only building and not serving. Please try running in production mode on your machine and confirm

Reply:
Now serving production build.